scenarios/tracking/02_mot_challenge.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<p>Why run this line "[seq.strip() for seq in train_seqs_str.split()] " if we can simply change this here to a list os strings (rather than the multi-line comment)</p><pre class="ql-syntax" spellcheck="false">test_seqs_str = """MOT17-01-SDP
                   MOT17-03-SDP
                   MOT17-06-SDP
                   MOT17-07-SDP
                   MOT17-08-SDP
                   MOT17-12-SDP
                   MOT17-14-SDP"""
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>